linux/guest_customization/gosc_support_matrix.yml

```diff
@@ -97,6 +97,17 @@ Pardus GNU/Linux:
       8.0.1: N/A
       8.0.2: 22385739
 
+Pardus GNU/Linux Server:
```

Comment:
Please update the comment for Pardus GNU/Linux XFCE and Server

Reply:
Change comment. Thanks